fitbenchmarking/fitbenchmarking
# 602 optional dependencies

#### Description of Work

Fixes #602 


#### Testing Instructions

1. Check optional install works (e.g., using Docker containers)
2. Check docs
3. Check tests pass

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
